docs/guided-tour.md

```diff
@@ -56,29 +56,45 @@ During this tutorial, you're going to try out some cool features of Sandstorm. Y
 
 One of the common reasons to use Sandstorm is to have online collaboration software that runs on your own server.
 
-Etherpad is a real-time text editor that demonstrates how to use Sandstorm that way. Make sure you're logged into Sandstorm (if using the demo, click **Start the demo**). Then click:
+Etherpad is a real-time text editor that demonstrates how to use Sandstorm that way. Make sure you're logged into Sandstorm (if using the demo, click **Try a quick demo**). Then:
 
-* **Install apps** - this takes you to the Sandstorm app list. (Soon to be the Sandstorm App Marketplace!)
-* Click the **Install** button next to _Etherpad_. (Sandstorm may ask you for confirmation.)
-* Click **New Etherpad Document** button on your Sandstorm.
+* Click **Install...** - this takes you to the Sandstorm App Market.
+* Find _Etherpad_ and click the **Install** button. Sandstorm will ask you for confirmation.
+* Click **Install Etherpad**.
+* Once Etherpad is installed, click **Create new pad**.
 
 You're now in an Etherpad document. Installing Etherpad allows you to create as many Etherpad documents as you want.
 
-If you want to create spreadsheets on your server, consider installing _Ethercalc_.
+If you want to create spreadsheets on your server, consider installing [_Ethercalc_](https://apps.sandstorm.io/app/a0n6hwm32zjsrzes8gnjg734dh6jwt7x83xdgytspe761pe2asw0).
```

Comment:
If these are going to be links, I'm -0 on having them italicized, but I'm A-OK with merging this as-is, but I thought I should at least express that feeling somewhere.